components/tab-tile/tab-tile.js

```diff
@@ -88,28 +35,138 @@ function scrollToElement(tab) {
   const contentElementBounding = contentElement.getBoundingClientRect()
   if (tabRowBounding.top <= 0 && contentElementBounding.bottom >= 0) {
-    contentElement.style.paddingTop = tabRowBounding.height + 'px'
+    contentElement.style.paddingTop = 40 + 'px'
     return true
   } else {
     contentElement.style.paddingTop = 0
     return false
   }
 }
+
+function locateTabsToCenter() {
+  for (let i = 0; i < tabContainers.length; i++) {
+    const numberOftabsAndSpacers = tabContainers[i].childElementCount
+    if (numberOftabsAndSpacers <= 4) {
+      applyTabsCentered(tabContainers[i].parentElement)
+    }
+  }
+}
+
+function applyTabsCentered(tabContainer) {
+  tabContainer.classList.add('msds-tabs--center-tabs')
+}
+
+function applyTabActiveEvent() {
+  for (let i = 0; i < tabs.length; i++) {
+    const tab = tabs[i]
+    const tabIndex = i
+    tab.addEventListener('click', () => setActive(tab, tabIndex))
+  }
+}
+
+function applyTabKeyboardNavigation() {
+  document.body.onkeyup = function(e) {
+    if (e.keyCode == 13) {
+      for (let i = 0; i < tabs.length; i++) {
+        const tab = tabs[i]
+        const tabIndex = i
+        const tabHasFocus = document.activeElement
+        if (tabHasFocus === tab) {
+          setActive(tab, tabIndex)
+        }
+      }
+    }
+  }
+}
+
+function applyTabsSticky() {
+  if (stickyTabsContainers) {
+    window.addEventListener(
+      'scroll',
+      function() {
+        for (let i = 0; i < stickyTabsContainers.length; i++) {
+          const currentStickyTabsContainer = stickyTabsContainers[i]
+          const stickyTabRow = currentStickyTabsContainer.querySelector('.msds-tabs__container')
+          const tabContent = currentStickyTabsContainer.querySelector('.msds-tabs__content-container')
+          const shouldAddStickyClass = hasElementHitTop(stickyTabRow, tabContent)
+          if (shouldAddStickyClass) {
+            stickyTabRow.classList.add('msds-tabs--sticky-tabs')
+          } else {
+            stickyTabRow.classList.remove('msds-tabs--sticky-tabs')
+          }
+        }
+      },
+      false
+    )
+  }
+}
+
+function ensureCardsContentMatchingHeights() {
+  const cardRows = document.querySelectorAll('.card-row')
+
+  cardRows.ensureCardContentsSameHeightWithinParent({
+    subSelectors: ['.msds-category-card__header-body', '.msds-category-card__content-body']
+  })
+
+  cardRows.ensureCardContentsSameHeightWithinParent({
+    subSelectors: [
+      '.msds-contact-card__headline',
+      '.msds-contact-card__body-contact-person',
+      '.msds-contact-card__body-contact-position',
+      '.msds-contact-card__body-contact-details'
+    ]
+  })
+
+  cardRows.ensureCardContentsSameHeightWithinParent({
+    subSelectors: [
+      '.msds-solution-card__header-headline',
+      '.msds-solution-card__body-headline',
+      '.msds-solution-card__body-description',
+      '.msds-solution-card__footer-buttons'
+    ]
+  })
+}
+
+function init() {
+  locateTabsToCenter()
```

Comment:
For these:

applyTabActiveEvent()
applyTabKeyboardNavigation()
applyTabsSticky()

you can assign events in init function on create bindEvents() function for that purpose. It has a few benefits: 
1. It helps read a code, since you see that part of the code is executed immedietely (locateTabsToCenter, applySmallTabs) and rest when event is fired.
2. It helps understand logic, since on left side you have elements which interact with user and names of events. On right side you have a name of functions when events are fired.
3. If your functions what handle the events and interactive elements are called in logic way it's easy to understand logic what stands behind your entire js. For example:

editButton.onclick = openEditModal
createButton.onclick = openCreateModal
tab.onclick = setTabActive

Or alternativly:

editModalButton.onclick = handleEditModalButtonClick
createButton.onclick = handleCreateModalButtonClick
tab.onclick = handleTabClick

I prefer first way but if you have a lot of events it can be that second way can be more readable. Please notice that these assignment create kind of mapping between dom elements, events and your event functions. These few lines of code can be like a table of contents of book. You start read book from them and they help you to understand what is going in your js. You can easly undertand that on this button when user click modal is open or tab become active. You even don't have to read a rest of code.